tests/foreman/api/test_setting.py

```diff
@@ -47,7 +53,7 @@ def test_positive_update_login_page_footer_text(self):
                 self.assertEqual(login.value, login_text)
 
     @run_in_one_thread
-    @tier1
+    @tier2
```

Comment:
Since this is functional test on acceptance level, I think it should be on Tier3 (according to test metadata doc, Tier2 is for functional tests on Component or Integration level).

Reply:
I even thought about it making at tier3, did not have a more clear thought.

Reply:
Look like I will support my comment, as while reviewing the code I can see that only single entity is been used. So keeping :CaseLevel: Acceptance does not make sense

Reply:
Yes, functional test case on Component level definitely does belong to Tier2. It's good now.